.github/workflows/npm-publish.yml

```diff
@@ -47,6 +47,28 @@ jobs:
         id: package
         uses: andreigiura/action-nodejs-package-info@v1.0.2
 
+      - name: Create GitHub Release
```

Comment:
should we do this automatically or manually ?
if we do it manually we can see who made the release